### PR TITLE
Add userinfo errors

### PR DESCRIPTION
**Pull request by Sgtpluck**
Jira ticket:
https://cm-jira.usa.gov/browse/LG-11487

This PR adds the userinfo errors to the error dictionary.
The errors can be found here: https://github.com/18F/identity-idp/blob/main/config/locales/openid_connect/en.yml#L47-L50

### CODE REVIEW

**Review by Jeremy1026 — APPROVED** [review]

File: _data/errors.yml [hunk 1]
```diff
@@ -125,7 +125,7 @@ oidc:
       accordion-id: token_expired
       content: |
         ##### Why it's happening
-        In the authentication response, a `code` value is returned that is then used in the `token` endpoint request. "The code is intended to be used within 15 minutes immediately after it is generated. This error may mean that the user paused for too long during the authentication attempt.
+        In the authentication response, a `code` value is returned that is then used in the `token` endpoint request. The code is intended to be used within 15 minutes immediately after it is generated. This error may mean that the user paused for too long during the authentication attempt.
```
Comment (nprimak) [inline _data/errors.yml:128]: I know this wasn't part of your changes but while we are here I would say 
`The code is intended to be used within 15 minutes after it is generated`
To reduce chances of confusion but I defer to @karenstrother  on language

File: _data/errors.yml [hunk 2]
```diff
@@ -195,3 +195,31 @@ oidc:
...
         - Ensure that the public certificate that matches the private key used to sign the JWT is registered in your application's configuration in the [Partner Dashboard](https://dashboard.int.identitysandbox.gov/){:target="_blank"}.
+  userinfo:
+    - title: Malformed Authorization header
+      id: malformed-auth-header
+      accordion-id: malformed_auth_header
+      content: |
+        ##### Why it's happening
+        The `userinfo` endpoint requires an Authorization Header that includes a Bearer token. The Bearer token value is the `access_token` value from the `token` endpoint response.
+        ##### What to do:
+        - Ensure you are sending an [Authorization header](https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Authorization){:target="_blank"} in your request to the `userinfo` endpoint.
```
Comment (nprimak) [inline _data/errors.yml:206]: Can you also add `{:class="usa-link--external"}` after the url to indicate that it is an external url? Also for all of the other mozilla urls like on line 215 and 224

**Review by nprimak — APPROVED** [review]
looks good, some minor comments

File: _data/errors.yml (hunk 1 above)
Comment (mitchellhenke) [reply _data/errors.yml:128]: I think the language could be stronger too. It's more than an intention, the code _will_ expire.

Comment (Sgtpluck) [reply _data/errors.yml]: updated in https://github.com/18F/identity-dev-docs/pull/398/commits/4bb0d783af1e2c2a834fc8fcaadfe47a09c24370

File: _data/errors.yml (hunk 2 above)
Comment (Sgtpluck) [reply _data/errors.yml]: added in https://github.com/18F/identity-dev-docs/pull/398/commits/4bb0d783af1e2c2a834fc8fcaadfe47a09c24370

File: _data/errors.yml (hunk 1 above)
Comment (karenstrother) [reply _data/errors.yml]: This is on me - I suggested the change to "The code is intended to be used within 15 minutes immediately after it is generated."

**Pull request merged**